recipes/qt/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+libX11-devel
+libXt-devel
+libXext-devel
+libXrender-devel
+gtk2-devel
+dbus-devel
+libXi-devel
+mesa-libGLU-devel
```

Comment:
They are right for CentOS 6.7 (which I'm using). 

I'm not clear if gtk2-devel is actually needed - if we install it does that mean that users will need gtk2 installed also?